### PR TITLE
Added a check for disabled cookies

### PR DESCRIPTION
**Pull request by nitw-shobhit**
https://trello.com/c/p4Nbf3nm/2483-signin-page-constantly-refreshed-if-cookies-disabled

### CODE REVIEW

File: orcid-web/src/main/webapp/static/javascript/script.js [hunk 1]
```diff
@@ -220,31 +220,39 @@ function myTest() {
...
-                        // detects server is down or CSRF mismatches
-                        // do to session expiration or server bounces 
-                        function() {
-                            console.log("error with loggin check on :"
-                                + window.location.href);
-                            window.location.reload();
-                    });
+	
+	//Test if cookies are enabled
+	OrcidCookie.setCookie("cookieTest", "test", 1);
```
Comment (rcpeters) [inline orcid-web/src/main/webapp/static/javascript/script.js]: We should test for a already existing cookie.

**Comment by nitw-shobhit** [comment]
@rcpeters updated the pull request.

File: orcid-web/src/main/webapp/static/javascript/script.js [hunk 2]
```diff
@@ -4106,4 +4102,11 @@ function isIndexOf(needle) {
         };
     }
     return indexOf.call(this, needle);
-};
\ No newline at end of file
+};
+
+function checkIfCookiesEnabled() {
+	OrcidCookie.setCookie("cookieTest", "test", 1);
```
Comment (rcpeters) [inline orcid-web/src/main/webapp/static/javascript/script.js]: @nitw-shobhit  Let's make this a function of OrcidCookie so you have to call OrcidCookie.checkIfCookiesEnabled(). Looking good!

**Comment by nitw-shobhit** [comment]
@rcpeters done.

**Pull request merged**